woocommerce/woocommerce-android
# Handle storage of stats granularity and custom range for stats and top performers independently

Closes: #11237
Follow up from: https://github.com/woocommerce/woocommerce-android/pull/11361


### Description

These changes allow for stats view and top performers view to operate completely independently from each other when changing stats granularity or setting a new custom date range. 

### Testing instructions

1. Enable dynamic dashboard feature flag
2. Launch the app
3. Play around with top performers view and check that: 
   - The stats granularity is independent between store stats and top performers
   - The custom range is independent between store stats and top performers
   - Killing the app will correctly restore stats granularity and custom range for both views.
